packages/odyssey/src/scss/base/_elements.scss

```diff
@@ -0,0 +1,86 @@
+blockquote {
+  padding: 0 $base-spacing;
+  border-left: 3px solid $base-border-color;
+}
+
+cite {
+  font-style: italic;
+}
+
+del {
+  display: inline-block;
+  background: cv('danger', 'lightest');
+}
+
+del::before,
+del::after {
+  position: absolute;
```

Comment:
dose the mixin not work for this and `ins`?

Reply:
it does! has been added.